api/tasks/runner.go

```diff
@@ -172,6 +199,9 @@ func (t *task) run() {
 		t.fail()
 		return
 	}
+	var user db.User
+	t.fetch("User not found", &user, "select * from user where id=?", t.task.UserID)
+	util.UserVaultCache.DecrementUsage(user.Username)
```

Comment:
no error checking

Reply:
if a failure does not need to stop program execution then please at least log it.